src/Jackalope/Version/VersionHandler.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Jackalope\Version;
+
+use Jackalope\Session;
+use Jackalope\Transport\AddNodeOperation;
+use Jackalope\Transport\WritingInterface;
+use PHPCR\InvalidItemStateException;
+use PHPCR\NodeInterface;
+use PHPCR\PropertyType;
+use PHPCR\UnsupportedRepositoryOperationException;
+use PHPCR\Util\UUIDHelper;
+use PHPCR\Version\VersionException;
+
+class VersionHandler
+{
+    const MIX_VERSIONABLE = 'mix:versionable';
+    const MIX_SIMPLE_VERSIONABLE = 'mix:simpleVersionable';
```

Comment:
Do I have to fix this in order for this PR to be merged? Or can we wait with that until `NodeType::isNodeType` has been enhanced?